ares/n64/aleck64/io.cpp

```diff
@@ -0,0 +1,59 @@
+auto Aleck64::readWord(u32 address, Thread& thread) -> u32 {
+  if(address <= 0xc07f'ffff) {
+    return sdram.read<Word>(address);
```

Comment:
Needs mask too, but haven't run into game that uses it yet
```suggestion
    return sdram.read<Word>(address & 0x00ff'ffff);
```